src/elements/common/nav-button/NavButton.js

```diff
@@ -26,6 +27,7 @@ type Props = {
 
 const NavButton = React.forwardRef<Props, React.Ref<any>>((props: Props, ref: React.Ref<any>) => {
     const {
+        'data-testid': dataTestId,
```

Comment:
Here as well, the `..rest` captures all the props that aren't explicitly listed.

Reply:
Have to explicitly list 'data-testid' here because it's a string, which needs to be converted to a valid variable name.

Reply:
Never mind, I noticed that I can totally remove it from this layer.